docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -416,6 +456,14 @@ Assuming you are using [`Uncomplicated Firewall`](https://en.wikipedia.org/wiki/
 sudo ufw allow from **YOUR_BLOCK_PRODUCER_INTERNAL_IP** to any port **YOUR_RELAY_LISTENING_PORT** proto tcp
 ```
 
+:::tip
+Here is a example of the command to run :
+
+```bash
+sudo ufw allow from 123.123.123.123 to any port 3128 proto tcp
```

Comment:
`123.123.123.123` should be a real IP that you define in the set of parameters above